client/src/plugins/vuetify.js

```diff
@@ -10,7 +10,8 @@ vuetifyConfig.theme.options.customProperties = true;
 vuetifyConfig.theme.dark = true;
 vuetifyConfig.theme.themes.dark = {
   ...vuetifyConfig.theme.themes.dark,
-  ...{ accent: colors.blue.lighten3 }
+  ...{ accent: colors.blue.lighten1 },
+  ...{ primary: "#004787" }
```

Comment:
why spread these objects?

Reply:
was copying over what was there, but looking closer at it, seems unneeded.  I'll remove.